### PR TITLE
Add examples of common category nodes

### PR DESCRIPTION
**Pull request by HiroyasuNishiyama**
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Proposed changes

This PR add examples of *common* category nodes. 

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [x] For non-bugfix PRs, I have discussed this change on the forum/slack team.
- [x] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

File: packages/node_modules/@node-red/nodes/examples/common/debug/1 example.json [hunk 1]
```diff
@@ -0,0 +1 @@
...","x":450,"y":360,"wires":[]},{"id":"2048c1f6.e72bee","type":"comment","z":"3ae4b3d9.1f77bc","name":"Output complete object","info":"","x":180,"y":420,"wires":[]},{"id":"fe9b300e.bdb96","type":"inject","z":"3ae4b3d9.1f77bc","name":"","topic":"Sample","payload":"Hello, World!","payloadType":"str","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":240,"y":460,"wires":[["5b7d830c.00bdbc"]]},{"id":"5b7d830c.00bdbc","type":"debug","z":"3ae4b3d9.1f77bc","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"true","targetType":"full","x":430,"y":460,"wires":[]},{"id":"d923a725.4ff058","type":"comment","z":"3ae4b3d9.1f77bc","name":"Output complete object","info":"","x":180,"y":520,"wires":[]},{"id":"e97d46cb.65fdb8","type":"inject","z":"3ae4b3d9.1f77bc","name":"","topic":"Sample","payload":"Hello, World!","payloadType":"str","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":240,"y":560,"wires":[["9441873.2c7ef78"]]},{"id":"9441873.2c7ef78","type":"debug","z":"3ae4b3d9.1f77bc","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"\"[\" & topic & \"] \" & payload","targetType":"jsonata","x":440,"y":560,"wires":[]}]
```
Comment (dceejay) [inline packages/node_modules/@node-red/nodes/examples/common/debug/1 example.json:1]: on final example - the comment box should probably have a more descriptive title - "Formatting debug using JSONata" or something like that ?

File: packages/node_modules/@node-red/nodes/examples/common/link/1 example.json [hunk 2]
```diff
@@ -0,0 +1 @@
...be6fc","name":"","links":["a30d20e6.ec6dc"],"x":315,"y":180,"wires":[]},{"id":"353c85ce.993d0a","type":"inject","z":"2f2d9fa4.be6fc","name":"","topic":"","payload":"Hello, World!","payloadType":"str","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":210,"y":180,"wires":[["70d6f012.8fe6d"]]},{"id":"6bf52c5c.d301c4","type":"debug","z":"2f2d9fa4.be6fc","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"false","x":370,"y":240,"wires":[]},{"id":"62ea32aa.d73aac","type":"comment","z":"2f2d9fa4.be6fc","name":"Example: Link Node","info":"Output of link out node can be connected to input of link in node.  The connection between links in/out is not shown, so the flow representation can be simplified.","x":130,"y":40,"wires":[]},{"id":"85133fcc.e482","type":"comment","z":"2f2d9fa4.be6fc","name":"Link output of inject node to input of debug node","info":"","x":260,"y":100,"wires":[]},{"id":"c588bc36.87fec","type":"comment","z":"2f2d9fa4.be6fc","name":"↓ connect to link in node","info":"","x":410,"y":140,"wires":[]},{"id":"8abca900.6dfe78","type":"comment","z":"2f2d9fa4.be6fc","name":"↑ connect from link out node","info":"","x":340,"y":280,"wires":[]}]
```
Comment (dceejay) [inline packages/node_modules/@node-red/nodes/examples/common/link/1 example.json:1]: Should probably have an extra comment box saying that link nodes can link across multiple tabs. (or pages)

File: packages/node_modules/@node-red/nodes/examples/common/debug/1 example.json (hunk 1 above)
Comment (HiroyasuNishiyama) [reply packages/node_modules/@node-red/nodes/examples/common/debug/1 example.json]: changed comment title according to your suggestion.

File: packages/node_modules/@node-red/nodes/examples/common/link/1 example.json (hunk 2 above)
Comment (HiroyasuNishiyama) [reply packages/node_modules/@node-red/nodes/examples/common/link/1 example.json]: added 2nd example that connect link nodes across tabs.

**Comment by knolleary** [comment]
The Debug example contains 5 different example flows - we should try not to do that. Each example should be demonstrating one thing and allow the user to get the example they need without having to import one example and then search through it.

**Comment by HiroyasuNishiyama** [comment]
updated example flow to make each example individual one

**Comment by HiroyasuNishiyama** [comment]
added missing examples of inject node referring related Node-RED cookbook items.

**Pull request merged**